config/clusters/openscapes/staging.values.yaml

```diff
@@ -1,8 +1,13 @@
 basehub:
```

Comment:
I think this can now go and together with it all the hardcoded special-cases we have all over the place 😅 yuhuuu 🚀